Sources/HaCWebsiteLib/Controllers/WorkshopsController.swift

```diff
@@ -23,4 +23,11 @@ struct WorkshopsController {
     }
   }
 
+  static var workshopUpdateHandler: RouterHandler = { request, response, next in
+    NewWorkshopManager.update()
```

Comment:
Does this perform an update or queue one? If the former, note that Heroku will always time out if a request takes more than 30s (but updates shouldn't take that long?)

Reply:
This performs one. Will change this

Reply:
Actually this was by design so that the user can load this page, and then go reload the workshops pages to see their changes right away. Maybe should wait and see if this is an issue in practice? I honestly am not sure how long updates will practically take.

Reply:
Okay let's fly and see how this does.